manifests/hipchat.pp

```diff
@@ -0,0 +1,34 @@
+# == Class: sensu_handlers::hipchat
+#
+# Sensu handler for sending to hipchat using hipchat gem
+#
+# == Parameters
+#
+# [*api_key*]
+#
+#  hipchat api key
+class sensu_handlers::hipchat (
+  $api_key,
+  $default_room = false,
```

Comment:
Can you remove this aspect of this handler? There is no analogous counterpart to this in any other of the handlers. He had something similar where alerts would default to "operations" and we quickly found that it was a bad idea.

Build for team-centric alerting from the beginning here.